lib/hocon/version.rb

```diff
@@ -0,0 +1,5 @@
+module Hocon
+  module Version
+    STRING = '1.2.0'
```

Comment:
nice, I really like this... I was playing around with having multiple versions of the gem installed on the same ruby the other day, and trying to figure out which one was getting loaded.  this will be convenient to have in the future.